### DISCUSSION TITLE
50~150 connections, a bulk insert with long insert Query

### DISCUSSION THREAD

**Discussion by fengzk-edgeray** [taosdata/TDengine#7554] [Q&A]
![image](https://user-images.githubusercontent.com/89452329/130720056-ab516fe0-703c-4005-95e4-b7d6938585c8.png)
![image](https://user-images.githubusercontent.com/89452329/130720364-d74b398e-3524-4763-856e-3922c1240539.png)
![image](https://user-images.githubusercontent.com/89452329/130597269-03cac099-305c-4f94-8281-2d43911c6fd1.png)
Background: Golang/taoSql/taos_connect fail()
Hello, td expert. this database was adopted to collect monitor status in our project, it inserted success but also happen many taos_connect fail()! error and 
How can I solve this problem?

**Comment by yu285** [comment]
we need you provide TDengine version,go driver version , and the program to us .

we will try to reproduce it

**Comment by aloscha1** [comment]
> ![image](https://user-images.githubusercontent.com/89452329/130720056-ab516fe0-703c-4005-95e4-b7d6938585c8.png)
> 
> ![image](https://user-images.githubusercontent.com/89452329/130720364-d74b398e-3524-4763-856e-3922c1240539.png)
> 
> ![image](https://user-images.githubusercontent.com/89452329/130597269-03cac099-305c-4f94-8281-2d43911c6fd1.png)
> 
> Background: Golang/taoSql/taos_connect fail()
> 
> Hello, td expert. this database was adopted to collect monitor status in our project, it inserted success but also happen many taos_connect fail()! error and 
> 
> How can I solve this problem?

**Comment by aloscha1** [comment]
To resolve the "taos_connect fail()" error:

1. Check network connectivity.
2. Verify server configuration.
3. Increase timeout settings if needed.
4. Monitor server resources.
5. Check server logs for clues.
6. Update client library.
7. Seek support from TDengine community.


Good Luck
